g/glance/build_info.json

```diff
@@ -2,17 +2,17 @@
   "maintainer": "vinodk99",
   "package_name": "glance",
   "github_url": "https://github.com/openstack/glance",
-  "version": "yoga-eom",
+  "version": "zed-eom",
   "default_branch": "master",
   "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
   "package_dir": "g/glance",
-  "build_script": "glance_ubi_8.7.sh",
+  "build_script": "glance_ubi_9.3.sh",
   "validate_build_script": "true",
   "use_non_root_user": "false",
   "docker_build": "true",
-  "yoga-eom": {
-    "dir": "yogaeom_ubi8",
-    "build_script": "glance_ubi_8.7.sh"
+  "*-eom": {
+    "dir": "zedeom_ubi9",
+    "build_script": "glance_ubi_9.3.sh"
   },
   "*.*.*": {
     "dir": "yogaeom_ubi8",
```

Comment:
default should go for new dir and script here @vinodk99

Reply:
@nikhil-kalbande updated the build_info.json file as suggested